src/components/views/settings/devices/DeviceDetails.tsx

```diff
@@ -18,13 +18,17 @@ import React from 'react';
 
 import { formatDate } from '../../../../DateUtils';
 import { _t } from '../../../../languageHandler';
+import AccessibleButton from '../../elements/AccessibleButton';
 import Heading from '../../typography/Heading';
 import { DeviceVerificationStatusCard } from './DeviceVerificationStatusCard';
 import { DeviceWithVerification } from './types';
 
 interface Props {
     device: DeviceWithVerification;
     onVerifyDevice?: () => void;
+    // @TODO(kerry) optional while signout only implemented
+    // for current device (PSG-744)
```

Comment:
what's the deal with this?

Reply:
I'm implementing signing out of current session separately to signing out of other sessions since the code paths are very different. This component is generic and used by both current and other sessions, so while I haven't added signout code to the other sessions this is an optional prop and sign out is not rendered when it's falsy.

I'm working on the other sessions half right now, so this will go away in my next PR